### ISSUE TITLE
SharePoint NewsFeed Integration

### ISSUE THREAD

**Issue by chri55w** [chri55w/QuickBooks-Timer#22]
Add integration with the Sharepoint Newsfeed for easier access to view and update the Newsfeed, with the potential to add reminders and notifications etc. 

**Useful Links**

https://msdn.microsoft.com/en-us/library/office/jj163977.aspx

**Comment by chri55w** [comment]
@BradConnor Thoughts on this as a future development (once this is properly released and tested) ?

**Comment by BradConnor** [comment]
Yes. I'm hoping that we can use this new timesheet system to ultimately remove QuickBooks from the mix altogether, apart from for billing - i.e. integration to Newsfeed, maybe even Q-Pulse ultimately (we can log effort against support calls and projects, maybe we could match and insert values from the timesheet system). I'd like to be able to create monthly activity reports etc. direct from a single, shared timer database, too. Not sure how best to implement the shared timer DB, though!
